src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,11 @@
+@use '../../styles/variables' as var;
+@use '../../styles/mixins' as mix;
+
+.footer {
+  flex-shrink: 0; /* Udrží footer dole */
+  padding: 1rem;
+  background-color: var.$secondary-color;
+  text-align: center;
+  @include mix.heading3;
+  position: relative;
```

Comment:
Position should be first. I won't comment on the ordering if it won't be absolutely random. Keep the ordering in mind, and try to do it by yourself :), during self code review.